app/role.go

```diff
@@ -240,7 +242,11 @@ func (a *App) CheckRolesExist(roleNames []string) *model.AppError {
 
 func (a *App) sendUpdatedRoleEvent(role *model.Role) {
 	message := model.NewWebSocketEvent(model.WebsocketEventRoleUpdated, "", "", "", nil)
-	message.Add("role", role.ToJson())
+	roleJSON, jsonErr := json.Marshal(role)
+	if jsonErr != nil {
+		mlog.Warn("Failed to encode role to JSON")
```

Comment:
Nit pick: Should `jsonErr` also get logged?

Reply:
Ideally yes, I'll try to update these.